### PR TITLE
blockchain: error on v2 selection

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

Remove v2 flag from toml

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -961,7 +961,7 @@ func (cfg *FastSyncConfig) ValidateBasic() error {
 	case BlockchainV0:
 		return nil
 	case BlockchainV2:
-		return nil
+		return errors.New("fastsync version v2 is not supported")
```
Comment (cmwaters) [inline config/config.go:964]:
```suggestion
		return errors.New("fastsync version v2 is no longer supported. Please use v0")
```

**Review by cmwaters — APPROVED** [review]
LGTM

**Review by cmwaters — COMMENTED** [review]
This also covers the case when someone tries to programatically instantiate a tendermint node right?

File: config/config.go (hunk 1 above)
Comment (marbar3778) [reply config/config.go:964]:
```suggestion
		return errors.New("fastsync version v2 is no longer supported. Please use v0")
```

**Comment by marbar3778** [comment]
> This also covers the case when someone tries to programatically instantiate a tendermint node right?

added an error in the node.go

**Review by cmwaters — APPROVED** [review]

**Pull request merged**